server/models/Collective.js

```diff
@@ -1258,6 +1259,19 @@ Collective.prototype.deactivateAsHost = async function () {
     );
   }
 
+  // Make sure we clean up all pending applications
+  await models.HostApplication.update(
+    { status: HostApplicationStatus.EXPIRED },
+    { where: { HostCollectiveId: this.id } },
+  );
+
+  await models.Member.update({ deletedAt: new Date() }, { where: { MemberCollectiveId: this.id, role: 'HOST' } });
+
+  await models.Collective.update(
+    { HostCollectiveId: null },
+    { hooks: false, where: { HostCollectiveId: this.id, isActive: false } },
+  );
+
```

Comment:
Not needed, we do this already in deactivateBudget: https://github.com/opencollective/opencollective-api/blob/dc25778c2ae332f27891f632d7fd09fc0ba9f7c1/server/models/Collective.js#L1410

```suggestion
```